lang/c++/include/avro/Node.hh

```diff
@@ -170,7 +170,8 @@ public:
         doAddCustomAttribute(customAttributes);
     }
 
-    virtual CustomAttributes getCustomAttributes() const = 0;
```

Comment:
Changing the return type an existing member function is a backward incompatible change and my break existing client code. I suggest that we leave the existing function as such and name the new function something like `customAttributeCount()`.

Reply:
It is not a breaking change because that member function was added last week: https://github.com/apache/avro/pull/3325